pages/dkp/konvoy/2.2/Install/index.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: layout.pug
+navigationTitle: Install
+title: Install
+menuWeight: 30
+excerpt: Install a DKP cluster
+beta: false
+enterprise: false
+---
+
+When installing DKP for a project, line-of-business, or enterprise, the first step is to determine the infrastructure on which you want to deploy.
+
+For example, you can:
+
+- Install on a public cloud infrastructure, such as Amazon Web Services (AWS) or Azure.
+- Install on an internal network on-premises with a physical (bare metal) or virtual infrastructure.
+- Install on air-gapped Edge.
+
+<p class="message--important"><strong>IMPORTANT: </strong>The infrastructure you select then determines the specific requirements for a successful installation.</p>
+
+
+DKP is a tool for provisioning Kubernetes clusters with a suite of pre-selected [Cloud Native Computing Foundation (CNCF)][cncf] and community-contributed tools.
+By combining a native Kubernetes cluster as its foundation with a default set of cluster extensions,
+DKP provides a complete out-of-the-box solution for organizations that want to deploy production-ready
+Kubernetes.
+
+As an example, this Installation guide provides simplified instructions to get your DKP cluster up and running with minimal configuration requirements on an Amazon Web Services (AWS) public cloud instance. For information about installing on a different platform, see [Choose Infrastructure](../choose-infrastructure/). 
+
+# Before you begin
+
+Before installing DKP, ensure you have the [following](../supported-operating-systems/).
+
+# Installing DKP
+
+1.  Install required packages. In most cases, you can install the required software using your preferred package manager. For example, on a macOS computer, you can use [Homebrew][brew] to install `kubectl` and the `aws` command-line utility by running the following command:
+
+    ```bash
+    brew install kubernetes-cli awscli
+    ```
+
+1.  Check the Kubernetes client version. Many important Kubernetes functions **do not work** if your client is outdated. You can verify that the version of `kubectl` you have installed is supported by running the following command:
+
+    ```bash
+    kubectl version --short=true
+    ```
+
+1.  To download DKP, see the [Download Konvoy](../download) topic for information. You will need to download and extract the DKP binary package tarball.
+
+1.  Verify you have valid **CLOUD PROVIDER security credentials** to deploy the cluster on that platform. This step is not required if you are installing DKP on an on-premises environment. For information about installing in an on-premises environment, see [Install on-premises](../choose-infrastructure/on-prem).
+
+
```

Comment:
![image](https://user-images.githubusercontent.com/3445370/165844624-3edb3f00-c924-46be-bad3-103799a70b5d.png)

Numbering is not contiguous. If it should be,  please remove this line.
```suggestion
```

Reply:
Fixed it!  Good catch!